### ISSUE TITLE
[survival beta ] Two types of crystal: confusing

### ISSUE THREAD

**Issue by Uncle-Ulty** [tsunamayo/Starship-EVO#5493]
On my first trip to the asteroid, I "mined" the crystals. When I returned to the assembler, I couldn't build the gravity unit, somehow the game didn't accept the crystal I had.

So I went to the asteroid again. This time I gathered the "natural form" crystal, and returned to the assembler...

The "gathered" and "mined" crystals are different, and one of them is a craft material, but the other isn't. It's kinda confusing.

**Comment by tsunamayo** [comment]
@Uncle-Ulty Yeah, it is because you should not be able to pick up the crystal, it is a bug.
